tests/test_helpers.py

```diff
@@ -344,7 +344,8 @@ def test_ipv6_addresses() -> None:
 
 def test_host_addresses() -> None:
     hosts = [
-        "www.four.part.host" "www.python.org",
+        "www.four.part.host",
+        "www.python.org",
```

Comment:
This is a good example of implicit concatenation being misleading

Before, this would have been unintentionally testing against a string value of `"www.four.part.hostwww.python.org"`